pylibui/libui/separator.py

```diff
@@ -0,0 +1,52 @@
+"""
+ Python wrapper for libui.
+
+"""
+
+import ctypes
+from . import clibui
+
+
+class uiSeparator(ctypes.Structure):
+    """Wrapper for the uiSeparator C struct."""
+
+    pass
+
+
+def uiSeparatorPointer(obj):
+    """
+    Casts an object to uiSeparator pointer type.
+
+    :param obj: a generic object
+    :return: uiSeparator
+    """
+
+    return ctypes.cast(obj, ctypes.POINTER(uiSeparator))
+
+
+# - uiSeparator *uiNewHorizontalSeparator(void);
+def uiNewHorizontalSeparator():
+    """
+    Returns an horizonal separator.
```

Comment:
This should be *__a__ horizontal separator*. Am I wrong? http://dictionary.cambridge.org/dictionary/english/horizontal